post_processing_stages/meson.build

```diff
@@ -103,7 +77,7 @@ endif
 enable_hailo = false
 hailort_dep = dependency('HailoRT', modules : ['HailoRT::libhailort'], version: '>=4.17.0',
                          required : get_option('enable_hailo'))
-hailo_tappas_dep = dependency('hailo-tappas-core', required : get_option('enable_hailo'))
+hailo_tappas_dep = dependency('hailo_tappas', required : get_option('enable_hailo'))
```

Comment:
I forget why I took some of this out of the build the other day but I do know I ran into `hailo-tappas-core` actually being called `hailo_tappas` when I built it from source.